### DISCUSSION TITLE
Loose your edit when using IOS devices

### DISCUSSION THREAD

**Discussion by ThorkilG12** [community/community#38208] [Other Feature Feedback, Questions, & Ideas]
**Hi**
Within the last year or so, something has heppend to the methods used to prevent users from loosing edits.
From a PC based browser you will see something like "Are you sure you want to leave this page. Unsaved edits" or whatever the browser says.
It's might be the javascript `beforeunload()` function or other methods used that are in action.

But... If you are on a browser on an IOS device no warning is given. You can edit something - close the browser - and nothing happend... Except that you have lost whatever you have edited.

Please help me, by firstly confirm that you see the same situation, and secondly... tada... Do you know how to avoid loosing your edits on IOS devices ?

The above problem is actually not related to GitHub as such...  
Everyone making an input field on a webpage will notice that IOS don't care about your edits.

But I assume that GitHub might be **very** interested in solving this.

--oOo--

**Comment by ossanna16** [comment]
Hi there @ThorkilG12 and welcome to our community! Thank you for the feedback 🙂
